harness/determined/common/storage/_shutil.py

```diff
@@ -0,0 +1,565 @@
+# flake8: noqa
```

Comment:
This is a lot to vendor in.  Can you get away with just copying the v3.8 `copytree` and `_copytree`, and changing some things like `copy2` to `shutil.copy2` (using the regular v3.7 `shutil` module, that is)?

Ideally, I'd love for this copy/paste job to be in the `storage/shared.py`, since that file is the maximum scope where this hack should be used.

Reply:
Fixed as recommended